models/src/main/java/com/vimeo/networking2/AlbumList.kt

```diff
@@ -1,14 +1,28 @@
 package com.vimeo.networking2
 
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
 import com.vimeo.networking2.common.Pageable
 
 /**
  * List of albums that could be paged.
  */
+@JsonClass(generateAdapter = true)
 data class AlbumList(
+
+    @Json(name = "total")
```

Comment:
I can't think of a way to de-duplicate these `@Json` annotations without making an abstract class and then subclassing it with the type, e.g. `AlbumList : AbstractList<Album>()`. I guess it might not be a big deal though.